adapters/sinatra/example/server.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+require 'rubygems'
+require 'bundler/setup'
+
+require 'sinatra/base'
+require 'opentelemetry/sdk'
+require_relative '../lib/opentelemetry/adapters/sinatra'
+
+# Set preferred tracer implementation:
+SDK = OpenTelemetry::SDK
+
+factory = OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
+factory.add_span_processor(
+  SDK::Trace::Export::SimpleSpanProcessor.new(
+    SDK::Trace::Export::ConsoleSpanExporter.new
+  )
+)
+
+OpenTelemetry::Adapters::Sinatra.install(name: 'sinatra-example', version: '1.0')
```

Comment:
The `config` hash isn't used anywhere. It seems confusing to specify `name` and `version` here when they're not used in anyway and don't actually mean anything.

Reply:
Updated to avoid passing `name` and `version` :heavy_check_mark: